### PR TITLE
[NeedsReview] B06902010 陳家潁

### PR DESCRIPTION
**Pull request by SparkleSouL**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is there any suggestion?

<br />

### CODE REVIEW

File: src/students/B06902010.py [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+        * Use len() to get the length of the list.
+        * Different data types could not be added. Use str() to convert data
+        to string.
+        * Run `python src/autograder.py -task 2 -student_id <your student ID>`
+        to see if you pass this task.
+        * The correct output would be (5, '1 taiwan')
+    '''
+    # TODO: change length and sentence to fit the requirement
+    length = len(input_list)
+    sentence = "1" + input_dictionary["a"]
```
Comment (amjltc295) [inline src/students/B06902010.py]: Hard-coding is not a good solution.

File: src/students/B06902010.py [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+    # TODO: download the image from img_url with the request module
+    from PIL import image
+    import requests
+    from io import BytesIO
+    import sys
+    sys.path.append("..")
+    from utils import draw_text
+    respond = requests.get(img_url)
+    result_img = Image_open(ByesIO(respond.content))
+    draw_text(result_img, 'b06202007')
```
Comment (amjltc295) [inline src/students/B06902010.py]: This is obviously plagiarism ...

Comment (SparkleSouL) [reply src/students/B06902010.py]: 我沒有筆電..好像是上傳錯檔案了

Comment (SparkleSouL) [reply src/students/B06902010.py]: 可以讓我重傳一次嗎

Comment (SparkleSouL) [reply src/students/B06902010.py]: Done

File: src/students/B06902010.py (hunk 1 above)
Comment (SparkleSouL) [reply src/students/B06902010.py]: Done

File: src/students/B06902010.py (hunk 2 above)
Comment (amjltc295) [reply src/students/B06902010.py]: Could you explain why there is other's student ID?

Comment (SparkleSouL) [reply src/students/B06902010.py]: 我沒有筆電然後剛剛在教同學怎麼寫,因為我自己在家clone一直出問題所以都還沒做，但我看過每個task了所以大概知道要怎麼寫，剛剛教完我同學之後因為task8比較多行就想說直接照著打結果忘記改id了，造成嫌疑真是對不起

**Pull request closed without merging**